src/runtime/math.py

```diff
@@ -217,6 +217,16 @@ def hypot(x: i32, y: i32) -> f64:
     """
     return sqrt(1.0*(x**2 + y**2))
 
+def trunc(x: f64) -> f64:
```

Comment:
Here the return type should be `i64`, right?
See:
https://github.com/lcompilers/lpython/blob/9ac009e68fd9139957de7385fa00f86afb8672f2/src/runtime/math.py#L34-L48
https://github.com/lcompilers/lpython/blob/9ac009e68fd9139957de7385fa00f86afb8672f2/src/runtime/math.py#L60-L73